tests/data_context/store/test_validation_config_store.py

```diff
@@ -0,0 +1,134 @@
+from __future__ import annotations
+
+from typing import TYPE_CHECKING
+from unittest import mock
+
+import pytest
+
+from great_expectations.core.batch_config import BatchConfig
+from great_expectations.core.data_context_key import StringKey
+from great_expectations.core.expectation_suite import ExpectationSuite
+from great_expectations.core.validation_config import ValidationConfig
+from great_expectations.data_context.cloud_constants import GXCloudRESTResource
+from great_expectations.data_context.store import ValidationConfigStore
+from great_expectations.data_context.types.resource_identifiers import GXCloudIdentifier
+
+if TYPE_CHECKING:
+    from tests.datasource.fluent._fake_cloud_api import CloudDetails
+
+
+@pytest.fixture
+def ephemeral_store():
+    return ValidationConfigStore(store_name="ephemeral_validation_config_store")
+
+
+@pytest.fixture
+def file_backed_store(tmp_path):
+    base_directory = tmp_path / "base_dir"
+    return ValidationConfigStore(
+        store_name="file_backed_validation_config_store",
+        store_backend={
+            "class_name": "TupleFilesystemStoreBackend",
+            "base_directory": base_directory,
+        },
+    )
+
+
+@pytest.fixture
+def cloud_backed_store(cloud_details: CloudDetails):
+    return ValidationConfigStore(
+        store_name="cloud_backed_validation_config_store",
+        store_backend={
+            "class_name": "GXCloudStoreBackend",
+            "ge_cloud_resource_type": GXCloudRESTResource.VALIDATION_CONFIG,
+            "ge_cloud_credentials": {
+                "access_token": cloud_details.access_token,
+                "organization_id": cloud_details.org_id,
+            },
+        },
+    )
+
+
+@pytest.fixture
+def validation_config() -> ValidationConfig:
+    return ValidationConfig(
+        name="my_validation",
+        data=BatchConfig(name="my_batch_config"),
+        suite=ExpectationSuite(name="my_suite"),
+    )
+
+
+@pytest.mark.parametrize("store_fixture", ["ephemeral_store", "file_backed_store"])
```

Comment:
why use strings for the stores and then call getfixturevalue? Can't you just reference the fixture directly?

Reply:
No, the fixture won't be properly instantiated unless you do it this way.
Fixtures only get loaded by pytest if you include them in test constructor.
Although this is a common complaint because it prevents you from using fixtures as part of `parametrize`.

There are packages out there like `pytest-lazy-fixture` that essentially do this work for you... but you still have to reference it by the string.
https://github.com/tvorog/pytest-lazy-fixture